### ISSUE TITLE
Support for .NETStandard 2.0

### ISSUE THREAD

**Issue by Mikilll94** [oxyplot/oxyplot-xamarin#82]
I am getting the following two warnings:

`Package 'OxyPlot.Core 1.0.0' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.	`

`Package 'OxyPlot.Xamarin.Forms 1.0.0' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.`

Are there any plans to support .NETStandard 2.0?

**Comment by xiaoyhe** [comment]
still not supported...

**Comment by saadsaifse** [comment]
Is there any update in this regard? I am still getting errors

```
NU1701: Package 'OxyPlot.Core 2.0.0-unstable0956' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
NU1701: Package 'OxyPlot.Xamarin.Forms 1.1.0-unstable0011' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
```

**Comment by objorke** [comment]
I have not checked this with Xamarin recently.
OxyPlot Core is currently targetting .NET Standard 1.0
https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/OxyPlot.csproj#L4
I think we should get a new stable release of OxyPlot.Core, then update OxyPlot.Xamarin.Forms

**Comment by bluekuen** [comment]
@objorke That would be great. :)

**Comment by gabrielgriesser** [comment]
Still no news ?

I'm working on a Xamarin.Forms project (only on Android) and realllllly need a stackedbar chart. Only OxyPlot has a free one, but this package doesn't support .NET Standard 2.0 :-(

**Comment by objorke** [comment]
Need help to solve this!

**Comment by dsyme** [comment]
It looks like this is done for the core assembly - do the others need to be netstandard2.0 as well? https://github.com/oxyplot/oxyplot-xamarin/blob/develop/Source/OxyPlot.Xamarin.Forms/OxyPlot.Xamarin.Forms.csproj#L4

**Comment by janusw** [comment]
We recently published a 2.0.0-beta.1 release, which is based on .NET Standard 2.0 (cf #125), so this issue is done:

https://github.com/oxyplot/oxyplot-xamarin/releases/tag/2.0.0-beta.1
